lib/components/ManageUI.jsx

```diff
@@ -57,8 +57,9 @@ export const ManageUI = (props) => {
           </ContentPane>
         )}
       </Content>
-      <div className='flex justify-center'>
+      <div className='flex justify-between w-3/4 sm:w-1/2 mx-auto'>
         <RelativeInternalLink link='/home'>View Prize Pool</RelativeInternalLink>
+        <RelativeInternalLink link='/admin'>Manage Pool</RelativeInternalLink>
```

Comment:
Why add this link rather than just make the `Admin` tab in `ManageUI` visible for all?

Reply:
Great question. I'm just not familiar with the ref app enough to know that's how the navigation works. 😁